src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -91,7 +92,16 @@ class FrameworkBundle extends Bundle
 {
     public function boot()
     {
-        ErrorHandler::register(null, false)->throwAt($this->container->getParameter('debug.error_handler.throw_at'), true);
+        if (class_exists(SymfonyRuntime::class)) {
+            $handler = set_error_handler('var_dump');
+            restore_error_handler();
+        } else {
+            $handler = [ErrorHandler::register(null, false)];
```

Comment:
When symfony/runtime is not found, the index.php file registers ErrorHandler only in debug mode, so we need to register it to not break prod mode.

Reply:
shouldn't we check whether it is already registered ? Maybe some devs have changed the index.php file.

And wouldn't this still cause issues with PHPUnit 11 then when booting a kernel during tests ?